### PR TITLE
Origin/feature/aws deploy use domain

### PR DESCRIPTION
**Pull request by danielmiu**
### Requirements for making a pull request

Thank you for contributing to our project! 

Please fill out the template below to help the project maintainers review it as fast as possible and include your contribution to the project.

### What does it fix?

Closes 

Please mention the main changes this PR brings.

### How has it been tested?

Please describe the tests that you ran to verify your changes.

### CODE REVIEW

File: aws_deploy/dns.tf [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  alias {
+    zone_id                = module.front-end.zone_id
+    name                   = module.front-end.dns
+    evaluate_target_health = false
+  }
+}
+
+resource "aws_acm_certificate" "cert" {
+  domain_name               = terraform.workspace == "production" ? local.domain_root : module.front-end_dns.fqdn
+<<<<<<< Updated upstream
```
Comment (mmircea16) [inline aws_deploy/dns.tf:49]: Is this the remnant of a merge conflict?

File: aws_deploy/dns.tf [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+  validation_method = "DNS"
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_route53_record" "cert_validation" {
+  count   = length(aws_acm_certificate.cert.domain_validation_options.*)
+<<<<<<< Updated upstream
```
Comment (RaduCStefanescu) [inline aws_deploy/dns.tf:81]: Same here.

File: aws_deploy/main.tf [hunk 3]
```diff
@@ -3,6 +3,57 @@ provider "aws" {
...
+
+  domain_api = "${var.workspace == "production" ? local.subdomains_api[0]
+  : var.workspace == "staging" ? local.subdomains_api[1] : local.subdomains_api[2]}"
+
+  domain_identitysrv = "${var.workspace == "production" ? local.subdomains_identitysrv[0]
+  : var.workspace == "staging" ? local.subdomains_identitysrv[1] : local.subdomains_identitysrv[2]}"
+
+  domain_postgres = "${var.workspace == "production" ? local.subdomains_postgres[0]
+  : var.workspace == "staging" ? local.subdomains_postgres[1] : local.subdomains_postgres[2]}"
+<<<<<<< Updated upstream
```
Comment (RaduCStefanescu) [inline aws_deploy/main.tf:42]: Same here.

File: aws_deploy/security.tf [hunk 4]
```diff
@@ -51,9 +51,12 @@ resource "aws_security_group" "intra" {
   tags = {
     Name = "${local.name}-intra"
   }
-}
 
+<<<<<<< Updated upstream
```
Comment (RaduCStefanescu) [inline aws_deploy/security.tf:55]: Same here.

File: aws_deploy/service.tf [hunk 5]
```diff
@@ -84,18 +94,23 @@ module "front-end" {
...
 
-  container_port        = 80
-  execution_role_arn    = aws_iam_role.ecs_execution.arn
-  image                 = var.IMAGE_FRONTEND
-  prefix                = local.name
+  container_port     = 80
+  execution_role_arn = aws_iam_role.ecs_execution.arn
+  image              = var.IMAGE_FRONTEND
+  prefix             = local.name
+<<<<<<< Updated upstream
```
Comment (RaduCStefanescu) [inline aws_deploy/service.tf:103]: Same here.

File: aws_deploy/service.tf [hunk 6]
```diff
@@ -106,12 +121,16 @@ module "api" {
...
 
-  container_port        = 80
-  execution_role_arn    = aws_iam_role.ecs_execution.arn
-  image                 = var.IMAGE_API
-  prefix                = local.name
+  container_port     = 80
+  execution_role_arn = aws_iam_role.ecs_execution.arn
+  image              = var.IMAGE_API
+  prefix             = local.name
+<<<<<<< Updated upstream
```
Comment (RaduCStefanescu) [inline aws_deploy/service.tf:130]: Same here.

File: aws_deploy/service.tf [hunk 7]
```diff
@@ -128,14 +147,25 @@ module "identitysrv" {
...
   ]
+  certificate_arn = aws_acm_certificate.identitysrv.arn
 
   container_port     = 80
+  #task_role_arn      = aws_iam_role.ecs_instance.arn
   execution_role_arn = aws_iam_role.ecs_execution.arn
   image              = var.IMAGE_IDENTITYSERVER
   prefix             = local.name
-  region             = var.region
+<<<<<<< Updated upstream
```
Comment (RaduCStefanescu) [inline aws_deploy/service.tf:157]: Same here.

File: aws_deploy/service/variables.tf [hunk 8]
```diff
@@ -58,3 +60,14 @@ variable "secrets" {
   type    = string
   default = "[]"
 }
+
+variable "instance_count" {
+  type    = number
+  default = 1
+}
+<<<<<<< Updated upstream
```
Comment (RaduCStefanescu) [inline aws_deploy/service/variables.tf:68]: Same here.

File: aws_deploy/variables.tf [hunk 9]
```diff
@@ -2,36 +2,43 @@ variable "az_count" {
...
   default = 2
 }
-variable "region" {
-  type = string
-  default = "eu-west-1"
-}
 
 variable "prefix" {
   type    = string
+<<<<<<< Updated upstream
```
Comment (RaduCStefanescu) [inline aws_deploy/variables.tf:8]: Here as well.

File: aws_deploy/variables.tf [hunk 10]
```diff
@@ -2,36 +2,43 @@ variable "az_count" {
...
   default = "code4romania/stamacasa-identityserver:develop"
 }
 variable "IMAGE_POSTGRES" {
   type = string
-  default = "postgres:default"
+  default = "postgres"
 }
-
+variable "region" {
+<<<<<<< Updated upstream
```
Comment (RaduCStefanescu) [inline aws_deploy/variables.tf:36]: Yup, here as well.

**Pull request closed without merging**